### PR TITLE
Form

### PR DESCRIPTION
**Pull request by Haywayaheadshot**
# In this milestone I;
> added a method that takes the first user in the DB.
> added forms to add new posts, add new comments, and likes.
> added a method for simulating the first user creating new posts

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
```
Comment (MeqdamAlqudah) [inline app/controllers/comments_controller.rb:1]: - [x] Please There should be a form in the views/posts/new.html.erb file which accepts input for creating posts I can see here you just have two erb files for showing all the posts and for showing the post.
image
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Haywayaheadshot 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
image


 ## Highlights
:+1: Readme file
:+1: no linter errors

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Haywayaheadshot** [comment]
@MeqdamAlqudah Thank you for reviewing my project. I will work on the post view asap

**Comment by Haywayaheadshot** [comment]
@MeqdamAlqudah I see that my project is approved on the dashboard. But I have added the changes. Please let me if I can merge or do more corrections.

Thank you

File: app/views/post/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<%= form_with url: post_index_path, model: Post do |form| %>
+  <%= form.label :title, 'Add a title for your post: ' %><br>
+  <%= form.text_field :title, placeholder: 'Title:' %> <br>
+  <%= form.label :text, 'Add text to your post:' %><br>
+  <%= form.text_area :text, placeholder: 'Text' %><br>
+  <%= form.submit "Create Post"%>
+<% end %>
```
Comment (MeqdamAlqudah) [inline app/views/post/new.html.erb:7]: -  [Optional] Please make sure that the buttons in your code  matches the design 
image

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @Haywayaheadshot , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image

## Highlights 
:+1: no linter errors 
:+1: Readme file 
:+1: app is working 
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Haywayaheadshot** [comment]
@MeqdamAlqudah Thank you for the review. I will work on the optional suggestions before merging

**Pull request merged**